Packs/Gatewatcher/Integrations/Gcenterv102/Gcenterv102.py

```diff
@@ -0,0 +1,1265 @@
+        """
+        response = self._delete(
+            endpoint=f"/api/ignore-lists/asset-names/{ignore_id}/"
+        )
+        if response.status_code == 204:
+            demisto.info(f"Delete an ignore asset {ignore_id} on GCenter {self.ip}: [OK]")
+        else:
+            raise GwAPIException(
+                f"Delete an ignore asset {ignore_id} on GCenter {self.ip}: [FAILED]",
+                response.text, response.status_code, response.reason
+            )
+
+    def del_ignore_mac_address(self, ignore_id: int) -> None:
+        """Delete an ignore mac address.
+
+        Args:
+            ignore_id: Ignore list identifier.
+
+        Raises:
+            GwAPIException: If status_code != 204.
+        """
+        response = self._delete(
+            endpoint=f"/api/ignore-lists/mac-addresses/{ignore_id}/"
+        )
+        if response.status_code == 204:
+            demisto.info(
+                f"Delete an ignore mac address {ignore_id} on GCenter {self.ip}: [OK]"
+            )
+        else:
+            raise GwAPIException(
+                f"Delete an ignore mac address {ignore_id} on GCenter {self.ip}: [FAILED]",
+                response.text, response.status_code, response.reason
+            )
+
+    def del_ignore_kuser_ip(self, ignore_id: int) -> None:
+        """Delete an ignore Kerberos ip.
+
+        Args:
+            ignore_id: Ignore list identifier.
+
+        Raises:
+            GwAPIException: If status_code != 204.
+        """
+        response = self._delete(
+            endpoint=f"/api/ignore-lists/kuser-ips/{ignore_id}/"
+        )
+        if response.status_code == 204:
+            demisto.info(
+                f"Delete an ignore kerberos ip {ignore_id} on GCenter {self.ip}: [OK]"
+            )
+        else:
+            raise GwAPIException(
+                f"Delete an ignore kerberos ip {ignore_id} on GCenter {self.ip}: [FAILED]",
+                response.text, response.status_code, response.reason
+            )
+
+    def del_ignore_kuser_name(self, ignore_id: int) -> None:  # noqa: E501
+        """Delete an ignore Kerberos username.
+
+        Args:
+            ignore_id: Ignore list identifier.
+
+        Raises:
+            GwAPIException: If status_code != 204.
+        """
+        response = self._delete(
+            endpoint=f"/api/ignore-lists/kuser-names/{ignore_id}/"
+        )
+        if response.status_code == 204:
+            demisto.info(
+                f"Delete an ignore kerberos username {ignore_id} on GCenter {self.ip}: [OK]"
+            )
+        else:
+            raise GwAPIException(
+                f"Delete an ignore kerberos username {ignore_id} on"
+                f" GCenter {self.ip}: [FAILED]",
+                response.text, response.status_code, response.reason
+            )
+
+    def send_malware(self, name: str, content: str) -> dict:
+        """Send file to the GScan malcore analysis.
```

Comment:
The difference between malware and malcore is not obvious.
Malcore is the engine on the GCenter appliance that analyses file to detect malware.
Maybe the command is misnamed ? The idea is to find out if a file is a malware.

Reply:
Got it. So maybe it would be worth adding that explanation to the docstring of the command - something like: 
"Send a file to the GScan malcore analysis to determine if it is malware".